docs/hello_nextflow/06_hello_config.it.md

```diff
@@ -0,0 +1,622 @@
+# Part 6: Hello Config
+
+<div class="video-wrapper">
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/IuDO2HeKvXk?si=tnXTi6mRkITY0zW_&amp;list=PLPZ8WHdZGxmXiHf8B26oB_fTfoKQdhlik" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+</div>
+
+/// caption
+:fontawesome-brands-youtube:{ .youtube } See the [whole playlist on the Nextflow YouTube channel](https://www.youtube.com/playlist?list=PLPZ8WHdZGxmXiHf8B26oB_fTfoKQdhlik).
+///
+
+Questa sezione esplorerà come impostare e gestire la configurazione della pipeline Nextflow in modo da poterne personalizzare il comportamento, adattarla a diversi ambienti e ottimizzare l'utilizzo delle risorse _senza modificare una sola riga del codice del flusso di lavoro stesso_.
+
+Ci sono diversi modi per farlo; qui useremo il meccanismo di file di configurazione più semplice e comune, il file `nextflow.config`. 
+Ogni volta che c'è un file chiamato `nextflow.config` nella directory corrente, Nextflow caricherà automaticamente la configurazione da esso.
+
+!!!nota
+
+ Tutto ciò che inserisci in `nextflow.config` può essere sovrascritto in fase di esecuzione fornendo le direttive di processo o i parametri e i valori pertinenti sulla riga di comando, oppure importando un altro file di configurazione, secondo l'ordine di precedenza descritto [qui](https://www.nextflow.io/docs/latest/config.html).
+
+In questa parte della formazione, utilizzeremo il file `nextflow.config` per illustrare i componenti essenziali della configurazione di Nextflow, quali direttive di processo, esecutori, profili e file di parametri.
+
+Imparando a utilizzare efficacemente queste opzioni di configurazione, puoi migliorare la flessibilità, la scalabilità e le prestazioni delle tue pipeline.
+
+---
+
+## 0. Warmup: verifica che Docker sia abilitato ed esegui il Hello Config workflow
+
+Innanzitutto, un rapido controllo. C'è un file `nextflow.config` nella directory corrente che contiene la riga `docker.enabled = <setting>`, dove `<setting>` è `true` o `false` a seconda che tu abbia lavorato o meno alla Parte 5 di questo corso nello stesso ambiente.
+
+Se è impostato su `true`, non è necessario fare nulla.
+
+Se è impostato su `false`, impostalo ora su `true`.
+
+```console title="nextflow.config" linenums="1"
+docker.enabled = true
+```
+
+Una volta fatto questo, verifica che il workflow iniziale funzioni correttamente:
+
+```bash
+nextflow run hello-config.nf
+```
+
+```console title="Output"
+ N E X T F L O W   ~  versione 24.10.0
+
+Lancio `hello-config.nf` [reverent_heisenberg] DSL2 - revisione: 028a841db1
```

Comment:
Keep in mind that nextflow logs like this one should be kept in the original language, as they aren't translated in a user's machine!